CRITICAL - 2021-11-13 14:43:46 --> Undefined property: App\Controllers\Auth::$load
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(29): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 29, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 14:51:17 --> Undefined property: App\Controllers\Auth::$load
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(29): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 29, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 14:57:09 --> Undefined property: App\Controllers\Auth::$load
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 32, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 15:00:03 --> Undefined property: App\Controllers\Auth::$load
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(35): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 35, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 15:01:46 --> Call to undefined function App\Controllers\get_instance()
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 15:03:56 --> Undefined variable: ci
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(53): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 53, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 15:04:56 --> Call to a member function item() on null
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 15:04:57 --> Call to a member function item() on null
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 15:07:09 --> Trying to get property 'config' of non-object
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(53): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 53, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 15:07:21 --> Call to a member function item() on null
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 15:07:54 --> Call to undefined method App\Controllers\Auth::microsoftconfig()
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 15:18:20 --> Class 'App\Controllers\oauth_client_class' not found
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 16:14:47 --> Undefined variable: login_by_username
#0 C:\xampp\htdocs\creaw\app\Views\Auth\index.php(10): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 10, Array)
#1 C:\xampp\htdocs\creaw\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\creaw\system\Common.php(1119): CodeIgniter\View\View->render('Auth/index', Array, true)
#3 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(199): view('Auth/index')
#4 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->login('')
#5 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-11-13 16:26:42 --> syntax error, unexpected '$data' (T_VARIABLE), expecting ',' or ';'
#0 C:\xampp\htdocs\creaw\system\Router\Router.php(195): CodeIgniter\Router\Router->autoRoute('/auth/login/')
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(793): CodeIgniter\Router\Router->handle('/auth/login/')
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 16:31:24 --> Class 'App\Controllers\oauth_client_class' not found
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 16:40:20 --> Class 'App\Libraries\Microsoft\oauth_client_class' not found
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 16:41:23 --> Class 'App\Libraries\Microsoft\oauth_client_class' not found
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 16:41:52 --> Class 'App\Libraries\Microsoft\oauth_client' not found
#0 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-11-13 20:51:02 --> Class 'App\Libraries\Microsoft\http_class' not found
#0 C:\xampp\htdocs\creaw\app\Libraries\Microsoft\oauth_client.php(1634): App\Libraries\Microsoft\oauth_client_class->SendAPIRequest('https://login.l...', 'POST', Array, NULL, Array, NULL)
#1 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(77): App\Libraries\Microsoft\oauth_client_class->Process()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-11-13 20:54:43 --> Class 'App\Libraries\http_class' not found
#0 C:\xampp\htdocs\creaw\app\Libraries\Microsoft\oauth_client.php(1634): App\Libraries\Microsoft\oauth_client_class->SendAPIRequest('https://login.l...', 'POST', Array, NULL, Array, NULL)
#1 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(77): App\Libraries\Microsoft\oauth_client_class->Process()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-11-13 20:55:28 --> Trying to get property 'emails' of non-object
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(96): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 96, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 21:02:00 --> Trying to get property 'emails' of non-object
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(96): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 96, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 21:08:55 --> error_log() expects parameter 1 to be string, array given
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'error_log() exp...', 'C:\\xampp\\htdocs...', 95, Array)
#1 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(95): error_log(Array)
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-11-13 21:09:16 --> error_log() expects parameter 2 to be int, array given
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'error_log() exp...', 'C:\\xampp\\htdocs...', 95, Array)
#1 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(95): error_log('User', Array)
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-11-13 21:09:50 --> error_log() expects parameter 3 to be a valid path, array given
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'error_log() exp...', 'C:\\xampp\\htdocs...', 95, Array)
#1 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(95): error_log('User', 1, Array)
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-11-13 21:11:24 --> error_log(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set()
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'error_log(): Fa...', 'C:\\xampp\\htdocs...', 95, Array)
#1 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(95): error_log('User', 1, 'D:/a/log.txt')
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-11-13 21:12:19 --> Trying to get property 'emails' of non-object
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(97): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 97, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 21:12:45 --> Trying to get property 'emails' of non-object
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(97): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 97, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 21:13:52 --> Trying to get property 'emails' of non-object
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(98): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 98, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 21:14:38 --> Trying to get property 'emails' of non-object
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(98): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 98, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 21:15:01 --> Trying to get property 'emails' of non-object
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(98): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 98, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-11-13 21:19:30 --> Use of undefined constant console - assumed 'console' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\creaw\app\Controllers\Auth.php(95): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 95, Array)
#1 C:\xampp\htdocs\creaw\system\CodeIgniter.php(918): App\Controllers\Auth->microsoft_login()
#2 C:\xampp\htdocs\creaw\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\creaw\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\creaw\index.php(46): CodeIgniter\CodeIgniter->run()
#5 {main}
